index.html

```diff
@@ -9730,9 +9730,8 @@ <h2>User prompts</h2>
 
 <p><a>User prompts</a> that are spawned
  from <a><code>beforeunload</code></a> event handlers,
- are <a>dismissed</a> implicitly upon <a>navigation</a>
- or <a>close window</a>,
- regardless of the defined <a>user prompt handler</a>.
+ are <a>dismissed</a> implicitly upon a <a data-lt="Navigate To">navigation</a>
+ or a <a>close window</a>, regardless of the defined <a>user prompt handler</a>.
```

Comment:
this makes things less clear.  `Navigate To` is a webdriver spec - (it is a POST) so can never be WebDriver Initiated - unless webdriver is calling itself?!  (in which case it would seem to be implementation specific what would or would not end up calling this (would back actually get some history then `Navigate To` in implenentation X and implementation Y just use some `back()` like functionaltity provided?